packages/pyroscope-flamegraph/src/Toolbar.module.css

```diff
@@ -5,6 +5,10 @@
   align-items: center;
 }
 
+.navbar button {
+  color: var(--ps-neutral-2);
```

Comment:
Would be nice to define that somewhat globally no? Since it's clear this color applies app-wide.